tox.ini

```diff
@@ -12,21 +12,21 @@ deps =
 
 commands=
     flake8 redo
-    py.test -v --cov=redo --cov-report term-missing --doctest-modules 
+    py.test -v --cov=redo --cov-report term-missing --doctest-modules --ignore decision.py
 
 [testenv:py37-black]
 deps =
     black
 commands =
-    black --check .
+    black --check . --include redo/.*\.pyi?$
 
 [flake8]
-exclude = .ropeproject,.tox
+exclude = .ropeproject,.tox,decision.py
```

Comment:
Hmm, I'm not sure the best way to work around python projects that assume that all _python_ within them are source code for that project.

Maybe it's not a bad thing? :thinking: 
Either way, for this PR, I'm just excluding `decision.py` :tada:

Reply:
I can see not wanting to include this in coverage, but why wouldn't we want black/flake8 run on it?

Reply:
flake8/black could be excluded due to new versions of each expecting fixups in these files, that upstream hasn't taken yet. Or that hasn't been imported into "this" repo yet. The mindset to me of excluding "3rd party" -- but any first party support for the library I think we still would want.